sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -308,19 +308,20 @@ def _workflow_with_pipelinerun(self, task_refs, pipeline, pipeline_template, wor
       }
     }
 
-    # Generate PodTemplate
```

Comment:
We probably want to keep/update this comment so that other developers know what this code block is doing.

Reply:
I updated with ```# Generate TaskRunSpec PodTemplate:s``` to state that it is on task level now instead :)